pkg/apiserver/statement/queries.go

```diff
@@ -214,10 +235,19 @@ func QueryPlans(
 
 func QueryPlanDetail(
 	db *gorm.DB,
+	cacheService *sysschema.CacheService,
 	beginTime, endTime int,
 	schemaName, digest string,
-	plans []string) (result Model, err error) {
-	fields := getAllAggrFields()
+	plans []string,
+) (result Model, err error) {
+	aService, err := newAggrService(db, cacheService)
```

Comment:
This is not a service. Service is something long running in the background. For example, Statement is a service that serve routers.

How about simply call it `helper`